scripts/get-ui-templates-for-lerna.js

```diff
@@ -0,0 +1,8 @@
+const {getUiTemplates} = require('./get-ui-templates');
+
+const getUiTemplatesForLerna = () =>
+  getUiTemplates()
+    .map((t) => `--scope ${t}`)
+    .join(' ');
+
+console.log(getUiTemplatesForLerna());
```

Comment:
That way, we only publish ui templates.

Another option would be to add a `npm:publish:template` script to template packages only. So we don't rely on the `--scope` option.